__tests__/order/order.query.test.js

```diff
@@ -1,13 +1,19 @@
 /* eslint-disable no-undef */
 const { gql } = require('@apollo/client');
-const client = require('../../utils/apollo-test-client');
 const { newOrder } = require('./order.variables');
+const { setupApp } = require('../helper-functions');
+jest.mock('../../modules/upload/upload.service');
 
 let orderId;
+let operations;
 
-describe('Order queries', () => {
+describe('Pattern queries', () => {
```

Comment:
why did you change `order` to `pattern`?

Reply:
Done